apps/mull-api/src/app/event/event.resolver.ts

```diff
@@ -17,6 +17,28 @@ export class EventResolver {
     return this.eventService.findOne(id);
   }
 
+  @Query(/* istanbul ignore next */ () => [Event])
+  async hostEvents(@Args('id', { type: /* istanbul ignore next */ () => Int }) id: number) {
+    return this.eventService.findHostEvents(id);
+  }
+
+  @Query(/* istanbul ignore next */ () => [Event])
+  async coHostEvents(@Args('id', { type: /* istanbul ignore next */ () => Int }) id: number) {
+    return this.eventService.findCoHostEvents(id);
+  }
+
+  @Query(/* istanbul ignore next */ () => [Event])
+  async participatingEvents(
+    @Args('id', { type: /* istanbul ignore next */ () => Int }) id: number
+  ) {
+    return this.eventService.findJoinedEvents(id);
+  }
+
+  @Query(/* istanbul ignore next */ () => [Event])
+  async discoverEvents(@Args('id', { type: /* istanbul ignore next */ () => Int }) id: number) {
+    return this.eventService.findDiscoverEvent(id);
+  }
```

Comment:
Minor problem but I feel that the working of the method and args are a bit vague as to what these functions do. Here's my suggestion, you can take parts of it or ignore it, but try to make these easier to understand since it's what is going to be called in the frontend.:
```suggestion
  async hostedEvents(@Args('userId', { type: /* istanbul ignore next */ () => Int }) id: number) {
    return this.eventService.findHostEvents(id);
  }

  @Query(/* istanbul ignore next */ () => [Event])
  async coHostedEvents(@Args('userId', { type: /* istanbul ignore next */ () => Int }) id: number) {
    return this.eventService.findCoHostEvents(id);
  }

  @Query(/* istanbul ignore next */ () => [Event])
  async participatingEvents(
    @Args('userId', { type: /* istanbul ignore next */ () => Int }) id: number
  ) {
    return this.eventService.findJoinedEvents(id);
  }

  @Query(/* istanbul ignore next */ () => [Event])
  async discoverEvents(@Args('userId', { type: /* istanbul ignore next */ () => Int }) id: number) {
    return this.eventService.findDiscoverEvent(id);
  }
```

Reply:
before I apply these changes, these methods are meant to return all the events of a co host or host. If the new naming implies that I will push the new changes

Reply:
Maybe the "ed" at the end wasn't right, I was trying to find a short wording to describe the behaviour of the method. On second thought the original names you had are okay because I can't think of something that's clearer but still short.

Reply:
Ragith and I couldn't think of any either and we were hoping someone would

Reply:
Yeah disregard this comment